cli/src/audit/detectSurrogates.ts

```
@@ -0,0 +1,383 @@
+    if (codePoint === undefined) {
+      // Should not happen with valid strings, but handle defensively.
+      i++;
+      continue;
+    }
+
+    result.push(`U+${codePoint.toString(16).toUpperCase().padStart(4, "0")}`);
+  }
+  return result.join(" ");
+
+
+  for (let i = 0; i < input.length; i++) {
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)